### ISSUE TITLE
Updating Login page and login card

### ISSUE THREAD

**Issue by Sailesh3000** [algovengers/MindMate#79]
Login page does not seem that interactive and attractive can u assign me this issue .I am a **GSSOC'24 Contributor** I will try my best to update it. Thank you

**Comment by subharthihazra** [comment]
Can you elaborate or show a sample of UI?

**Comment by Sailesh3000** [comment]
![image](https://github.com/algovengers/MindMate/assets/126142666/df9086de-e400-4a0f-acfe-6daccd66282b)
I can make this look like 
![image](https://github.com/algovengers/MindMate/assets/126142666/8ed0c734-2061-4b8a-a660-f78fd1136451)
Thats what I wanted to convey

**Comment by subharthihazra** [comment]
Yeah go on, keep the theme colors aligned with the rest of site kindly.

**Comment by Sailesh3000** [comment]
![image](https://github.com/algovengers/MindMate/assets/126142666/b9c3809f-ffb2-4eed-a689-dfb52831dcfb)
is this ok or should I update it

**Comment by Sailesh3000** [comment]
please review the above @subharthihazra
